.rubocop.yml

```diff
@@ -11,7 +11,7 @@ Layout/LineLength:
 Metrics/BlockLength:
   ExcludedMethods: ['describe', 'context', 'before']
 Metrics/MethodLength:
-  Max: 15
+  Max: 30
```

Comment:
we're disabling this all over the place so i've just bumped it up (lmk if i should do this in a separate PR--I realize it's making this one kind of messy).